source/manage-apps/deploy/compact-disk.txt

```diff
@@ -0,0 +1,66 @@
+.. _compact-disk:
+
+=======================
+Compact an Atlas Volume
+=======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+
+Overview
+--------
+
+While Realm data compaction defragments the storage space the realm file uses 
+on a *client*, and also reduces the size of the __realm_sync database on Atlas 
+clusters, it does not reduce the overall size of the Atlas cluster. Compacting 
+the data on an Atlas volume involves several additional steps. 
+
+Steps to Compact and Reclaim Disk Space
+---------------------------------------
+
+You should use the following guidance for reducing Atlas disk space usage:  
+
+1. Enable Compaction on the Client
+
+   Before reclaiming volume space, be sure that you have enabled Compaction 
+   on your Atlas App Services app. You set this by setting the Max Offline Time 
+   setting in the Atlas UI. For more information, see 
+   :ref:`<optimize-sync-atlas-usage>`.
+
+   .. note:: Wait for Compaction
+   
+      If you enabling compaction for the first time, you will need to wait for 
```

Comment:
```suggestion
      If you enable compaction for the first time, you will need to wait for 
```
Is this only the first time or any time you enable compaction?

Reply:
Technically it's any time, but a) if its already enabled, there should be no wait, and b) the recommendation in the ticket is to "wait up to a week" (!).